core/OrderAnalyser.py

```diff
@@ -40,11 +40,19 @@ def start(self):
 
         if not launched_synapses:
             Utils.print_info("No synapse match the captured order: %s" % self.order)
-        else:
-            for synapse in launched_synapses:
-                params = self._get_synapse_params(synapse, self.order)
-                for neuron in synapse.neurons:
-                    self._start_neuron(neuron, params)
+
+        if not launched_synapses and self.main_controller.settings.default_synapse is not None:
+            default_synapse = self._get_default_synapse(self.brain.synapses, self.main_controller.settings.default_synapse)
```

Comment:
python should fail to find the default_synapse as it is not declared in Settings model.